std_msgs/Header header
float32 x
float32 y
bool button_x
bool button_circle
bool button_triangle
bool button_square